Sources/_CSwiftSyntax/src/atomic-counter.c

```diff
@@ -1,3 +1,5 @@
+#include "atomic-counter.h"
+
 #include <stdint.h>
```

Comment:
Nitpick, this `#include` can be removed now.

Reply:
I was wondering if you'd ask me to remove that or not! I usually try not to rely on indirect inclusions for unrelated types, but since it's part of the signature in this case, it doesn't really matter. Done.